package.json

```diff
@@ -64,7 +63,7 @@
   "dependencies": {
     "axios": "^0.22.0",
     "axios-retry": "^3.2.0",
-    "eventsource": "^1.1.2",
+    "@harness/event": "file:../eventsource",
```

Comment:
This will be updated once the package is published to NPM. Feel free to mark the PR as requested changes until this is done, along with any other required changes.